cmd/node/factory/structs.go

```diff
@@ -1841,7 +1841,20 @@ func newMetaBlockProcessor(
 	rater sharding.PeerAccountListAndRatingHandler,
 ) (process.BlockProcessor, error) {
 
-	builtInFuncs := builtInFunctions.NewBuiltInFunctionContainer()
+	argsBuiltIn := builtInFunctions.ArgsCreateBuiltInFunctionContainer{
```

Comment:
this will cause conflicts with soft-restart feature branch (not having a great idea how to overcome this)